### PR TITLE
Cheatsheets: Updating CSS for new style via issue #1226

### PR DESCRIPTION
**Pull request by spentacular**
Sorry if my commit on pngs messed things up. I'm not quite sure why/how any of messed up with my initial pull.

### CODE REVIEW

**Comment by moollaza** [comment]
@spentacular the PNG's thing isn't your fault but it does make this PR difficult to merge. The problem is that the PNGs were written with case sensitive names and unfortunately not all operating systems accept case-sensitive filenames. We really need to make the names unique so this isn't an issue for anyone else.

**Comment by moollaza** [comment]
I pinged the Guitar Chords dev to fix that issue, but in the meantime I can cherry pick your commits to get this on a machine for testing :+1:

**Comment by moollaza** [comment]
@spentacular @abeyang changes are up on DDH7: https://ddh7.duckduckgo.com/?q=tmux+cheat+sheet&ia=cheatsheet

**Comment by abeyang** [comment]
Looks way, way better! Awesome! :+1:

**Comment by spentacular** [comment]
Thanks guys! It's all @abeyang though. Let me know if you have any low hanging CSS fruit, and I'd be more than happy to take care of it.

**Comment by abeyang** [comment]
May need your help, sooner than you think, @spentacular -- these changes break dark-mode. Will provide fixes soon! Thanks again for your help!

**Comment by spentacular** [comment]
Hey @abeyang I'll take a swing at this whenever you're ready for me!

File: share/goodie/cheat_sheets/cheat_sheets.css [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 .zci--cheat_sheets .cheatsheet__title {
     padding-bottom: 1em;
-    font-size: 1.1111em;
+    font-size: 1em;
     font-weight: bold;
-    color: #000;
+    color: #333;
```
Comment (moollaza) [inline share/goodie/cheat_sheets/cheat_sheets.css]: @spentacular we can actually remove this altogether! I just realized we already have `tx-clr--slate` in the template:https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/detail.handlebars#L8

**Comment by moollaza** [comment]
Closing. I believe all of this was merged in your other PR @spentacular ?

**Comment by spentacular** [comment]
Yep @moollaza !

**Pull request closed without merging**